src/twisted/protocols/test/test_tls.py

```diff
@@ -1163,38 +1171,56 @@ def test_producerDuringRenegotiation(self):
         If we write some data to a TLS connection that is blocked waiting for a
         renegotiation with its peer, it will pause and resume its registered
         producer exactly once.
+
+        Renegotiation only works on TLSv1.2 or less.
         """
-        c, ct, cp = self.setupStreamingProducer()
-        s, st, sp = self.setupStreamingProducer(server=True)
+        clientProtocol, clientTransport, _ = self.setupStreamingProducer()
+        serverProtocol, serverTransport, serverProducer = self.setupStreamingProducer(
+            server=True, serverMethod=TLSv1_2_METHOD
+        )
```

Comment:
The new locals names are better but reviewing this method was a bit trickier with both renaming and other changes mixed together, for what it's worth.

Reply:
Understood. 

I don't know what to say... there is a lot of overhear to create a ticket + pr + wait for tests.

Yes. Reviewing just the renaming would be easier for the reviewer ... but I would say that is a lot of wasted time for the PR author.

At the same time, I understand that with Twisted the review time is scarce.

I will try  to make as little change as possible and move typos/renaming in separate PRs.